src/pages/GE_TokenizeCard.page

```diff
@@ -13,56 +12,10 @@
 
     <script src="{!elevateSDKURL}"></script>
     <script>
-        const hostname = window.location.hostname;
-        const arr = hostname.split(".");
-        let domain = arr[0].replace('--npsp', '');
-        domain = domain.replace('--c', '');
-        const lightningOrigin = `https://${domain}.lightning.force.com`;
-
+        const sfdo = new sfdoPaymentsJsSdk({ debug: true });
+        const lightningOrigin = buildLightningOrigin();
         const DEFAULT_NAME_ON_CARD = '[Not Provided]';
-        let nameOnCard = DEFAULT_NAME_ON_CARD;
-
-        const sfdo = new sfdoPaymentsJsSdk();
-        sfdo.mount({
-            id: 'payments',
-            clientId: '{!productId}',
-            gatewayId: '{!gatewayIds}',
-            merchantId: '{!merchantIds}',
-            designSystem: 'Lightning',
-            styles: '' +
-                '.logo {' +
-                    'left: 8px !important;' +
-                '}' +
-                '.fieldset.cc-number {' +
-                    'padding-left: 0;' +
-                    'padding-right: 1rem;' +
-                '}' +
-                '.fieldset.exp {' +
-                    'padding-left: 0;' +
-                    'padding-right: 1rem;' +
-                '}' +
-                '.fieldset.cvc {' +
-                    'padding: 0;' +
-                '}' +
-                '.error-wrapper.fieldset.exp {' +
-                    'max-width: fit-content;' +
-                '}' +
-                '.error-wrapper.fieldset.cvc {' +
-                    'max-width: fit-content;' +
-                '}'
-        })
-        .then(() => {
-            isLoaded();
-
-            setTimeout(function() {
-                isLoaded();
-            }, 3000);
-        });
-
-        function isLoaded() {
-            const message = JSON.stringify({ isLoaded: true });
-            postMessage(message);
-        }
+        const postMessageType = 'post__npsp';
```

Comment:
If we are adding a `postMessageType` to our post messages, the Jest Tests in this [PR](https://github.com/SalesforceFoundation/NPSP/pull/6073) will need to be updated to respect post messages with this new type

Reply:
Yeah, this was one place where I was a bit ify. I'd like to get the dev team's temperature on this particular addition. It felt pretty safe to me since we're only generating postMessages from a limited number of places right now and we can cover all those to include this new property.